src/test/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/integration/wiremock/ExternalUsersApiMockServer.kt

```diff
@@ -61,9 +60,9 @@ class ExternalUsersApiMockServer : WireMockServer(WIREMOCK_PORT) {
     )
   }
 
-  fun stubCreateRole() {
+  fun stubCreateHttpStatus(url: String) {
```

Comment:
Would the name be better as stubPost() to match the others?

Reply:
Thanks Karen. Updated